output/archives.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
whole content of `output` directory shouldn't be tracked by git.

For more informations: https://git-scm.com/docs/gitignore (and before `git rm`: https://git-scm.com/docs/git-rm)